webpack.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  entry: "./source/index.js",
```

Comment:
No such file.

We need `./source/browser.js` and `./source/server.js`.

Create a new file called `cuid.js`. Make it export a function that takes an options object. Browser and Server should do this, respectively:

```
import createCuid from './cuid.js';
import fingerprint from './browser.js';

export default () => createCuid({ fingerprint });
```

```js
import createCuid from './cuid.js';
import fingerprint from './server.js';

export default () => createCuid({ fingerprint });
```